lib/global-styles.php

```diff
@@ -5,107 +5,6 @@
  * @package gutenberg
  */
 
-/**
```

Comment:
This has been moved verbatim to `WP_Theme_JSON::get_from_editor_settings`, except for the first 4 lines:

```php
$all_blocks                                = WP_Theme_JSON::ALL_BLOCKS_NAME;
$theme_settings                            = array();
$theme_settings['settings']                = array();
$theme_settings['settings'][ $all_blocks ] = array();
```

that were collapsed into one:

```php
$theme_settings = array( 'settings' => array() );
```